ui/src/features/project/pipelines/freight/duration-to-seconds.ts

```diff
@@ -0,0 +1,24 @@
+import { Duration } from 'date-fns';
+
+export const durationToSeconds = (duration: Duration): number => {
+  const {
+    years = 0,
+    months = 0,
+    weeks = 0,
+    days = 0,
+    hours = 0,
+    minutes = 0,
+    seconds = 0
+  } = duration;
+
+  const totalSeconds =
+    years * 365.25 * 24 * 60 * 60 +
+    months * 30.44 * 24 * 60 * 60 +
+    weeks * 7 * 24 * 60 * 60 +
+    days * 24 * 60 * 60 +
+    hours * 60 * 60 +
+    minutes * 60 +
+    seconds;
+
+  return Math.floor(totalSeconds);
+};
```

Comment:
We don't need this. Please use https://date-fns.org/v4.1.0/docs/differenceInMilliseconds

Reply:
Sorry I didn't get that. I don't need difference, I need seconds and not necessarily to find difference, besides I don't have exact Date because its duration, so I can't convert it without reference point either